config/theme.go

```diff
@@ -108,6 +108,161 @@ func NewBaseTheme(preset Preset) *Theme {
 			},
 		}
 
+	// Special Preset for Protanopias
+	case PresetProtDark:
+		return &Theme{
+			Default: MustParseColor("white"),
+			Base: ThemeBase{
+				Key:       MustParseColorSlice("#feb927 / white"),
+				Info:      MustParseColor("white"),
+				Primary:   MustParseColor("#4860e6"),             // magenta
+				Secondary: MustParseColor("#2aabee"),             // cyan
+				Success:   MustParseColor("#6afd6a:bold"),        // bold green
+				Warning:   MustParseColor("#feb927:italic"),      // yellow
+				Danger:    MustParseColor("fg=white:bg=#c2270a"), // red background
+				Muted:     MustParseColor("#feb927"),             // yellow
+			},
+			Data: ThemeData{
+				String: MustParseColor("#2aabee"),
+			},
+			Table: ThemeTable{
+				Header:  MustParseColor("white:bold"),
+				Columns: MustParseColorSlice("#2aabee / #6afd6a:bold / #4860e6 / white / #feb927"),
+			},
+			Options: ThemeOptions{
+				Flag: MustParseColor("#feb927"),
+			},
```

Comment:
Suggest just using the default fallback color here of secondary.

Reply:
Set to the secondary "cyan" color